### PR TITLE
Update dependencies etc.

### PR DESCRIPTION
**Pull request by Maxopoly**

### CODE REVIEW

**Comment by ProgrammerDan** [comment]
You cannot remove the CTP dependency: https://github.com/DevotedMC/BetterShards/blob/master/BetterShardsBukkit/src/main/java/vg/civcraft/mc/bettershards/listeners/CombatTagListener.java#L10

**Comment by Maxopoly** [comment]
Uh wait yeah, that wasnt really intended

**Comment by Maxopoly** [comment]
fixed

File: pom.xml [hunk 1]
```diff
@@ -26,7 +26,7 @@
 	<distributionManagement>
 		<repository>
 			<id>civcraft-repo</id>
```
Comment (ProgrammerDan) [inline pom.xml]: update name? :P

Comment (Maxopoly) [reply pom.xml]: Done

**Pull request merged**